internal/logging/logging.go

```diff
@@ -23,16 +30,44 @@ func initLogger() {
 		logLevel = logrus.InfoLevel
 	}
 
-	log.Level = logLevel
-	log.Out = os.Stdout
-	log.ReportCaller = true
+	if cfg.LogFormater == "text" {
+		logger.Formatter = &logrus.TextFormatter{}
+	} else {
+		logger.Formatter = &logrus.JSONFormatter{}
+	}
+
+	logger.Level = logLevel
+	logger.Out = os.Stdout
+	logger.ReportCaller = true
+
+	if cfg.CwAccessKey != "" {
```

Comment:
Would `viper.SetDefault("CwAccessKey", c.Logging.Cloudwatch.AccessKeyId)` set `cfg.CwAccessKey` to "" if not found?